Makefile

```diff
@@ -65,4 +69,4 @@ unit-in-ci:
 check-code:
 	yarn tsc-check
 	yarn eslint-check
-	yarn build
+	yarn build
```

Comment:
This line shouldn't be committed. Looks like your editor needs to be set to add new line at end of file